jaseci_serv/jaseci_serv/jac_api/tests/test_jac_api.py

```diff
@@ -1,9 +1,12 @@
 import base64
 import json
 
+from unittest.mock import MagicMock, Mock
```

Comment:
@marsninja when you look at this, you can ignore the change in this file -- this is for something else and not related to this bug.